src/app/fdctl/config/default.toml

```
@@ -279,6 +279,12 @@ dynamic_port_range = "8900-9000"
     # `--private-rpc` argument.
     private = false
 
+    # The address to bind the RPC port. If no address is specified,
```

nit: these comments are double spaced
```suggestion
    # The address to bind the RPC port.  If no address is specified,
```